.npmignore

```diff
@@ -0,0 +1,7 @@
+@bazel
```

Comment:
this seems wrong, don't users need this folder to get the starlark rules?

Reply:
nope they don’t need it. I have created these intermediary directories that are identical to what we get within npm_install repositories. since we don’t release this package as a starlark library, they don’t need this at all.